docs/releases/1.9.1.txt

```diff
@@ -64,3 +64,7 @@ Bugfixes
 
 * Restored the functionality of the admin's ``list_editable`` add and change
   buttons (:ticket:`25903`).
+
+* Fixed ``isnull`` query lookup for ``ForeignObject()``
```

Comment:
no parentheses for "ForeignObject" and no blank line at the end of the life please